pkg/git/git.go

```diff
@@ -34,3 +36,39 @@ func GetCommitHashes(repoPath string, commitLimit int) ([]string, error) {
 
 	return commitHashes, nil
 }
+
+// GetCommitTags returns the commit tags of a given repository ordered by the creation date. If `commitLimit` is -1 all tags will be returned.
+func GetCommitTags(repoPath string, tagLimit int) ([]string, error) {
+	var commitTags []string
+
+	repository, err := git.PlainOpen(repoPath)
+	if err != nil {
+		return nil, err
+	}
+
+	tagIter, err := repository.Tags()
```

Comment:
Do you know how the tags will be sorted?

This is crucial, as it defines which tags are deleted in the end. I.e. the commits are returned in commit time order, configured via the `git.LogOrderCommitterTime` option.

Reply:
I updated the ticket [VT-1071](https://ticket.vshn.net/browse/VT-1071) to reflect the new requirements (sort by version and in alphabetical order)

Reply:
As agreed a new flag has been introduced to resolve this issue